packages/react-core/src/components/ExpandableSection/ExpandableSection.tsx

```diff
@@ -109,6 +117,8 @@ export class ExpandableSection extends React.Component<ExpandableSectionProps, E
           propOrStateIsExpanded && styles.modifiers.expanded,
           isActive && styles.modifiers.active,
           isDetached && styles.modifiers.detached,
+          isDisplayLarge && styles.modifiers.displayLg,
```

Comment:
Everything looks good to me, but I'm wondering if we will ever add different 'display size' type props in the future? If we may plan to do that a `displaySize` prop with an enum may be more future proof

Reply:
@mcarrano wdyt about that - we use `display-large` as the name for the marketing/web variations of components. Do you anticipate we would have more variations of `display-[whatever]` in the future?

Reply:
Good question @kmcfaul .

Reply:
I guess if past history is a predictor, the only thing I could possibly think of is that we would get a request for a compact version with smaller text and tighter spacing.

Reply:
Those are typically called `pf-m-compact`, considering they're a regular variation of a component. A little background, one of the reasons we went with `.pf-m-display-large` was that we could have both a `.pf-m-large` variation as a regular modifier for a component intended for product use, and a `.pf-m-display-large` variation specifically for web/marketing use.

Reply:
I will make the prop a variant. I can do the same for the accordion PR.